ui/helpers/errorMessage.js

```diff
@@ -42,3 +42,18 @@ export const formatTaskclusterError = function formatTaskclusterError(e) {
 
   return `${TC_ERROR_PREFIX}${errorMessage}`;
 };
+
+/**
+Helper method for easily popping up an error message from Perfherder.
+
+@param {Error} e error object from Perfherder client.
+*/
+export function alertHttpError(error) {
+    if (error.statusText) {
+        error = 'HTTP Error: ' + error.statusText;
+    }
+    // we could probably do better than print this
+    // rather useless error, but at least this gives
+    // a hint on what the problem is
+    alert('Error loading performance data\n\n' + error);
```

Comment:
this helper method is called alertHttpError, but is specific to performance, the comments do help- so I don't know if we need to make this clearer that this is alertPerfherderHttpError

Reply:
I think the comments suffice.

Reply:
`alert()` is generally frowned upon for non-development, with in-page notifications preferred (eg thNotify, soon to be replaced by something react based for the job view). As such, I think this function should perhaps live in perfherder until its removed later when perfherder is converted to React?

@camd will have a better idea of the future direction, so I'll defer to him :-)

Reply:
I agree that this should just be left in perfherder for now. I think I only used this function as a stopgap to try and debug some problems.

Reply:
I put alertHttpError only in Perfherder. Once thNotify is refactored, I'll replace it with that.

Reply:
Agreed.  PR #4132 (currently in review) will replace ``thNotify`` with a React view component and a ``Context``.  So once you refactor this area to React, it should be easy to use.  For ``helpers`` we generally prefer to throw an exception that is then caught by the calling class which, in turn, reports to the user.